docs/dev/plugins.md

```diff
@@ -459,6 +459,7 @@ The `run_data` object contains all data from the Chef InSpec run. Here is an ove
 |`run_data.profiles[0].inputs`| Array of Input records describing inputs present in profile|
 |`run_data.profiles[0].inputs[0].name`| String name of Input|
 |`run_data.profiles[0].inputs[0].options.required`| Boolean, whether input was required.|
+|`run_data.profiles[0].inputs[0].options.sensitive`| Boolean, true if the value of the input has been replaced with "***" |
```

Comment:
This description might be confusing because the run_data from the runner has not yet redacted the sensitive value. That only happens in the base reporter.
So for someone developing reporter plugins, this description is 100% true; but if the reader is interested in the run_data API itself, or some other part of InSpec, this may be confusing because it says the value has been replaced with *** but they may see the real value in memory.

Reply:
Filtering happens to the Hash, then the Hash is converted to the RunData object. By the time the RunData object sees this value, it has indeed already been replaced.

This documentation is for the RunData object as seen by the reporter plugin system. The docs are in the file `docs/dev/plugins.md`, in the section regarding how to write a reporter plugin.